daemonize.go

```diff
@@ -152,7 +152,7 @@ func Run(
 	path string,
 	args []string,
 	env []string,
-	status io.Writer) (err error) {
+	status io.Writer, stderr *os.File) (err error) {
```

Comment:
Could you update the documentation, as in my comment in the main thread?

Reply:
Done